data-prepper-plugins/opensearch-source/src/main/java/org/opensearch/dataprepper/plugins/source/opensearch/configuration/SchedulingParameterConfiguration.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.opensearch.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
+import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
+import jakarta.validation.constraints.Min;
+import org.opensearch.dataprepper.parser.DataPrepperDurationDeserializer;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+
+public class SchedulingParameterConfiguration {
+
+    @JsonProperty("rate")
+    @JsonDeserialize(using = DataPrepperDurationDeserializer.class)
```

Comment:
This annotation is unneeded.

Reply:
You don't need either of those annotations. As noted previously, you should not depend on this class or `data-prepper-core`. The `data-prepper-core` project will provide this deserialization.